client/src/pages/Result.css

```diff
@@ -0,0 +1,4 @@
+.Result {
```

Comment:
Its generally best to go for lowercase class names

Reply:
Fixed